docs/en/week13/13-3.md

```diff
@@ -0,0 +1,518 @@
+train_acc: 0.5514, test_acc: 0.5900
+Epoch 6, train_loss: 1.6358, test_loss: 1.6405
+train_acc: 0.5800, test_acc: 0.6000
+Epoch 7, train_loss: 1.5748, test_loss: 1.5682
+train_acc: 0.6057, test_acc: 0.5600
+Epoch 8, train_loss: 1.5144, test_loss: 1.4885
+train_acc: 0.6686, test_acc: 0.6600
+Epoch 9, train_loss: 1.4497, test_loss: 1.4263
+train_acc: 0.7314, test_acc: 0.5900
+Epoch 10, train_loss: 1.3676, test_loss: 1.3474
+train_acc: 0.7286, test_acc: 0.5900
+Epoch 11, train_loss: 1.3136, test_loss: 1.2808
+train_acc: 0.7086, test_acc: 0.5900
+Epoch 12, train_loss: 1.2681, test_loss: 1.2135
+train_acc: 0.7257, test_acc: 0.6000
+Epoch 13, train_loss: 1.1674, test_loss: 1.1373
+train_acc: 0.7314, test_acc: 0.7000
+Epoch 14, train_loss: 1.1054, test_loss: 1.0750
+train_acc: 0.7486, test_acc: 0.7100
+Epoch 15, train_loss: 1.0271, test_loss: 1.0075
+train_acc: 0.7657, test_acc: 0.6800
+Epoch 16, train_loss: 1.0054, test_loss: 0.9638
+train_acc: 0.7543, test_acc: 0.7300
+Epoch 17, train_loss: 0.9375, test_loss: 0.9086
+train_acc: 0.7571, test_acc: 0.7500
+Epoch 18, train_loss: 0.8724, test_loss: 0.8526
+train_acc: 0.7714, test_acc: 0.7800
+Epoch 19, train_loss: 0.8415, test_loss: 0.8084
+train_acc: 0.7743, test_acc: 0.7300
+Epoch 20, train_loss: 0.7886, test_loss: 0.7648
+train_acc: 0.8000, test_acc: 0.7500
+Epoch 21, train_loss: 0.7142, test_loss: 0.7264
+train_acc: 0.8114, test_acc: 0.7900
+Epoch 22, train_loss: 0.7191, test_loss: 0.6993
+train_acc: 0.8029, test_acc: 0.7500
+Epoch 23, train_loss: 0.6859, test_loss: 0.6937
+train_acc: 0.8057, test_acc: 0.7400
+Epoch 24, train_loss: 0.6449, test_loss: 0.6228
+train_acc: 0.8314, test_acc: 0.8100
+Epoch 25, train_loss: 0.6372, test_loss: 0.6026
+train_acc: 0.8171, test_acc: 0.8400
+Epoch 26, train_loss: 0.5703, test_loss: 0.5927
+train_acc: 0.8429, test_acc: 0.7800
+Epoch 27, train_loss: 0.5655, test_loss: 0.5560
+train_acc: 0.8314, test_acc: 0.8400
+Epoch 28, train_loss: 0.5001, test_loss: 0.5227
+train_acc: 0.8943, test_acc: 0.8100
+Epoch 29, train_loss: 0.4841, test_loss: 0.5107
+train_acc: 0.8971, test_acc: 0.8500
+Epoch 30, train_loss: 0.5553, test_loss: 0.5520
+train_acc: 0.8086, test_acc: 0.8000
+Epoch 31, train_loss: 0.4682, test_loss: 0.4915
+train_acc: 0.8886, test_acc: 0.7800
+Epoch 32, train_loss: 0.4400, test_loss: 0.4347
+train_acc: 0.9114, test_acc: 0.8700
+Epoch 33, train_loss: 0.4211, test_loss: 0.4332
+train_acc: 0.9314, test_acc: 0.8200
+Epoch 34, train_loss: 0.4019, test_loss: 0.4137
+train_acc: 0.9371, test_acc: 0.8800
+Epoch 35, train_loss: 0.3648, test_loss: 0.4142
+train_acc: 0.9429, test_acc: 0.8900
+Epoch 36, train_loss: 0.3873, test_loss: 0.3748
+train_acc: 0.9114, test_acc: 0.9200
+Epoch 37, train_loss: 0.3478, test_loss: 0.3489
+train_acc: 0.9429, test_acc: 0.9400
+Epoch 38, train_loss: 0.3266, test_loss: 0.3475
+train_acc: 0.9543, test_acc: 0.9300
+Epoch 39, train_loss: 0.3885, test_loss: 0.3666
+train_acc: 0.9086, test_acc: 0.8700
+```
```

Comment:
Remove all of this. People have access to the notebook.
This is supposed to be a summary of the lecture.

Reply:
Done, extra code blocks have been cleaned